ethereum/contracts/nft/NFTBridge.sol

```diff
@@ -21,6 +21,8 @@ contract NFTBridge is NFTBridgeGovernance {
 
     // Initiate a Transfer
     function transferNFT(address token, uint256 tokenID, uint16 recipientChain, bytes32 recipient, uint32 nonce) public payable returns (uint64 sequence) {
+        require(evmChainId() == block.chainid, "invalid evmChainId");
```

Comment:
Should we create a function modifier for this check?

Reply:
I think this is a good idea to reduce duplicated code